### PR TITLE
OCA add: Percona PPM

### PR DESCRIPTION
**Pull request by hmorris3293**
##Description
A Percona PPM One-Click App

## Type of Change
 * Non breaking change ('update', 'change')
 
## Note to Reviewers
TODO:
 
 * [x]  Convert PNGs to SVGs
 * [x]  Set ordinal location of this OCA

### CODE REVIEW

File: packages/manager/src/features/OneClickApps/FakeSpec.ts [hunk 1]
```diff
@@ -382,6 +382,20 @@ export const oneClickApps: OCA[] = [
...
+    summary:
+      'An open source analytics and performance monitoring solution for databases with a focus on user-friendly metrics visualizations.',
+    related_info: [
+      {
+        title: 'Percona Monitoring and Management',
+        href:
+          'https://www.percona.com/software/database-tools/percona-monitoring-and-management'
+      }
+    ],
+    logo_url: 'assets/perconappm_color.svg'
```
Comment (Jskobos) [inline packages/manager/src/features/OneClickApps/FakeSpec.ts:397]: Typo preventing image load

File: packages/manager/src/features/OneClickApps/FakeSpec.ts [hunk 2]
```diff
@@ -382,6 +382,20 @@ export const oneClickApps: OCA[] = [
     ],
     logo_url: 'assets/plesk_color.svg'
   },
+  {
+    name: 'Percona PMM',
```
Comment (Jskobos) [inline packages/manager/src/features/OneClickApps/FakeSpec.ts:386]: I'm only 70% sure about this but I think we only check to see if the name of the FakeSpec entry matches the name of the StackScript. In which case this has to be "Percona" not "Percona PMM"

File: packages/manager/src/features/OneClickApps/FakeSpec.ts (hunk 1 above)
Comment (hmorris3293) [reply packages/manager/src/features/OneClickApps/FakeSpec.ts]: updated

File: packages/manager/src/features/OneClickApps/FakeSpec.ts (hunk 2 above)
Comment (hmorris3293) [reply packages/manager/src/features/OneClickApps/FakeSpec.ts]: updated

**Pull request merged**